crates/starknet-server/src/cli.rs

```diff
@@ -1,33 +1,71 @@
+use std::str::FromStr;
+
 use clap::Parser;
 use starknet_core::StarknetConfig;
+use starknet_in_rust::definitions::block_context::StarknetChainId;
 use starknet_types::num_bigint::BigUint;
 
 /// Run a local instance of Starknet Devnet
 #[derive(Parser, Debug)]
 #[command(author, version, about, long_about = None)]
 #[command(propagate_version = true)]
 pub(crate) struct Args {
-    /// Number of accounts
+    /// Number of accounts.
```

Comment:
I think we don't use `.` in other places. at the end

Reply:
Fixed in 796f389d0c66a19de306a91b29b072fccc9fb4f4